src/Evolution/Executables/ScalarWave/EvolveScalarWave.hpp

```diff
@@ -129,17 +130,21 @@ struct EvolutionMetavars {
       tmpl::conditional_t<local_time_stepping, tmpl::list<>,
                           dg::Actions::ApplyFluxes>,
       Actions::RecordTimeStepperData>>;
-  // To add filtering to the executable add the action:
-  //
-  // dg::Actions::ExponentialFilter<0,
-  //        tmpl::list<ScalarWave::Pi, ScalarWave::Psi, ScalarWave::Phi<Dim>>>
-  //
-  // to the end of `update_variables` so it is after `Actions::UpdateU`.
+
+  // Test filtering in 2D
```

Comment:
Could you make this into a stronger statement that filtering is likely not needed when evolving a 2D scalar wave? I suggest something like this:

"The Scalar Wave system generally does not require filtering, except perhaps on certain deformed domains. Here a filter is added in 2D for testing purposes. When performing numerical experiments with the Scalar Wave system, the user should determine whether this filter can be removed."